ui/src/app/pages/admin/workspace/admin-workspace.tsx

```diff
@@ -479,7 +480,7 @@ export class AdminWorkspaceImpl extends React.Component<Props, State> {
                   Runtime Name
                 </PurpleLabel>
                 <PurpleLabel style={styles.narrowWithMargin}>
-                  Google Project
```

Comment:
every runtime in this workspace shares the same google-project, and this info is displayed at the top of the page